pom.xml

```diff
@@ -18,7 +18,7 @@
         <java.version>1.8</java.version>
         <image.tag>latest</image.tag>
         <dependency.pdfbox.version>2.0.15</dependency.pdfbox.version>
-        <dependency.alfresco-core.version>7.3</dependency.alfresco-core.version>
+        <dependency.alfresco-core.version>7.14</dependency.alfresco-core.version>
         <dependency.alfresco-data-model.version>8.8</dependency.alfresco-data-model.version>
```

Comment:
Update alfresco-data-model (that is compatible with alfresco-core) ?

Reply:
I've updated both dependencies based on the latest ACS packaging pom.